arcgis-ios-sdk-samples/AppDelegate.swift

```diff
@@ -106,23 +106,29 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UISplitViewControllerDele
             navigationBarAppearance.configureWithTransparentBackground()
             navigationBarAppearance.largeTitleTextAttributes = [.foregroundColor: UIColor.white]
             navigationBarAppearance.titleTextAttributes = [.foregroundColor: UIColor.white]
-            navigationBarAppearance.backgroundColor = .primaryBlue
+            navigationBarAppearance.backgroundColor = .accentColor
             
             navigationBarAppearanceProxy.standardAppearance = navigationBarAppearance
             navigationBarAppearanceProxy.compactAppearance = navigationBarAppearance
             navigationBarAppearanceProxy.scrollEdgeAppearance = navigationBarAppearance
         } else {
             navigationBarAppearanceProxy.largeTitleTextAttributes = [.foregroundColor: UIColor.white]
             navigationBarAppearanceProxy.titleTextAttributes = [.foregroundColor: UIColor.white]
-            navigationBarAppearanceProxy.barTintColor = .primaryBlue
+            navigationBarAppearanceProxy.barTintColor = .accentColor
         }
+        UISwitch.appearance().onTintColor = .accentColor
         navigationBarAppearanceProxy.tintColor = .white
-        
-        UIToolbar.appearance().barTintColor = .backgroundGray
-        UIToolbar.appearance().tintColor = .primaryBlue
-        
-        UISwitch.appearance().onTintColor = .primaryBlue
-        UISlider.appearance().tintColor = .primaryBlue
+        if #available(iOS 14.0, *) {
+            // Nothing to do! iOS 14 handles global tint with accent color.
+        } else {
+            // Override tint color for `UIControl`s.
+            UIToolbar.appearance().tintColor = .accentColor
+            UISlider.appearance().tintColor = .accentColor
+            UITableViewCell.appearance().tintColor = .accentColor
+            UIProgressView.appearance().tintColor = .accentColor
+            UIView.appearance(whenContainedInInstancesOf: [UIAlertController.self]).tintColor = .accentColor
+            UIButton.appearance(whenContainedInInstancesOf: [AGSCallout.self]).tintColor = .accentColor
```

Comment:
@philium Instead of changing each storyboard which mapView that has a callout, I decide to override it in `AppDelegate`, as it involves less changes. What do you think?

Reply:
That works, although it probably doesn't need to be restricted to just `AGSCallout`. Either way is fine; we could always change it later if needed.

Reply:
@saratchandrakarumuri All issues you [mentioned](https://github.com/Esri/arcgis-runtime-samples-ios/pull/915#issuecomment-691335967) are resolved now! Feel free to take another look.